src/pet.js

```diff
@@ -0,0 +1,69 @@
+const MAXIMUM_FITNESS = 10;
+const ZERO = 0;
+const LOW_FITNESS = 3
+const HIGH_HUNGER = 5
+
+
+function Pet(name) {
+    this.name = name;
+    this.age = ZERO;
+    this.hunger = ZERO;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = []
+};
+
+Pet.prototype = {
+    get isAlive() {
+      return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+  };
+
+Pet.prototype.growUp = function () {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+      }
+        this.age += 1;
+        this.hunger += 5;
+        this.fitness -= 3;
+    };
+
+Pet.prototype.walk = function () {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+    } if ((this.fitness + 4) <= MAXIMUM_FITNESS ) {
+        this.fitness += 4;
+    } else {
+        this.fitness = MAXIMUM_FITNESS
+    }
+}
+
+Pet.prototype.feed = function () {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+    } if (this.hunger - 3 > ZERO) {
```

Comment:
makes code more readable to have if statements separated by line, as to not get confused with if else statements i.e. 
```
if (...) {
  ...
}

if (...) {
  ...
} else if (...) {
  ...
} else { 
 ...
}
```